main/Accordion/AccordionItem.tsx

```diff
@@ -40,37 +40,36 @@ type ToggleIndicatorType = React.ReactElement | undefined;
 interface TranslateYType {
   translateY: Animated.Value;
 }
+
 interface Props {
   testID: string;
+  styles?: Styles;
   datum: Datum;
-  shouldAnimate: boolean;
   collapseOnStart: boolean;
+  shouldAnimate: boolean;
   animDuration?: number;
   activeOpacity?: number;
   toggleElement?: React.ReactElement;
   dropDownAnimValueList: Animated.Value;
   sumOfPrecedingTranslateY: TranslateYType[];
   renderTitle?: (item: string) => React.ReactElement;
   renderBody?: (item: string) => React.ReactElement;
-  titleContainerStyle?: ViewStyle;
-  bodyContainerStyle?: ViewStyle;
 }
 
 const AccordionItem: FC<Props> = (props) => {
   const {
     testID,
     datum: item,
-    shouldAnimate,
-    collapseOnStart,
+    shouldAnimate = true,
+    collapseOnStart = true,
     animDuration,
     activeOpacity,
     toggleElement,
     dropDownAnimValueList,
     sumOfPrecedingTranslateY,
     renderTitle = (title) => <StyledTitle>{title}</StyledTitle>,
     renderBody = (body) => <StyledItem>{body}</StyledItem>,
-    titleContainerStyle,
-    bodyContainerStyle,
+    styles,
```

Comment:
How about giving `styles: {titleContainer, bodyContainer}`. This can lead to reducing test senarios.

Reply:
1. I don't understand what `This can lead to reducing test senarios.` mean. Can you kindly explain it?
2. `styles` is `Styles | undefined`. So I can not write `styles: {titleContainer, bodyContainer}` directly. I need to do `const {titleContainer, bodyContainer} = styles ?? {}`. Therefore I think using `?.` like now is better.

How do you think? 😃

Reply:
1. I think if you still need `2`, it won't reduce the test cases. However, to explain what I meant here in more detail, having `styles?.titleContainer`, we need to have test case for `styles.titleContainer` and when `styles` is undefined. Therefore, I was wondering if we can reduce this to just one single case.

Reply:
Also, `2` sounds better I think!

Reply:
@hyochan  Ok I got it.